plugins/linear-genome-view/src/LinearGenomeView/components/Highlight.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { useRef, useState } from 'react'
+import { observer } from 'mobx-react'
+import { makeStyles } from 'tss-react/mui'
+import { colord } from '@jbrowse/core/util/colord'
+import {
+  ParsedLocString,
+  Region,
+  SessionWithWidgets,
+  getSession,
+} from '@jbrowse/core/util'
+import { Menu } from '@jbrowse/core/ui'
+import { Base1DViewModel } from '@jbrowse/core/util/Base1DViewModel'
+import { IconButton, Tooltip, useTheme } from '@mui/material'
+
+// icons
+import LinkIcon from '@mui/icons-material/Link'
+import CloseIcon from '@mui/icons-material/Close'
+import BookmarkIcon from '@mui/icons-material/Bookmark'
+
+// locals
+import { LinearGenomeViewModel } from '../model'
+
+type LGV = LinearGenomeViewModel
+
+const useStyles = makeStyles()({
+  highlight: {
+    height: '100%',
+    position: 'absolute',
+    textAlign: 'center',
+    overflow: 'hidden',
+    display: 'flex',
```

Comment:
seems maybe to look the same without display:flex, textAlign,overflow:hidden, alignItems start? are they needed? can add comments if any are essential perhaps